src/main/java/com/cargohub/security/jwt/JwtTokenProvider.java

```diff
@@ -84,7 +84,7 @@ public boolean validateToken(String token) {
             }
 
             return true;
-        } catch (JwtException | IllegalArgumentException e) {
+        } catch (ExpiredJwtException e) {
```

Comment:
I suppose catching JwtException instead of  ExpiredJwtException will allow situations, when the token is not only expired but is also invalid, to be handled correctly.

Reply:
It was changed by mistake, thank you that you noticed. Changed back.